cl_sii/extras/dj_model_fields.py

```diff
@@ -115,8 +115,12 @@ def get_prep_value(self, value: Optional[Rut]) -> Optional[str]:
         However, these are preliminary non-DB specific value checks and
         conversions (otherwise customize :meth:`get_db_prep_value`).
 
+        Note: Before returning, ``value`` will be passed to :meth:`to_python` so that, if needed, it
+        will be converted to an instance of :class:`Rut`, which is very convenient in cases such
+        as when the type of ``value`` is :class:`str`.
         """
         value = super().get_prep_value(value)
+        value = self.to_python(value)
```

Comment:
Perhaps `value: Optional[Rut] = self.to_python(value)` would make things even clearer? It is quite easy to get confused with the mess of "DB value", "prep value", "python value", "serialized value"...

Reply:
Mypy reports the error `Name 'value' already defined on line 108` if I add a type annotation to that statement's `value` variable because the method argument `value` already has a type annotation (`value: Optional[object]`). I renamed `value` to `value_rut` so that the type annotation works.